From 78cfc3e85f1bc0d357beb81d30165ffa76c4d630 Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyfox@gentoo.org>
Date: Tue, 1 Jan 2013 15:18:11 +0300
Subject: [PATCH] Fix build failure against ghc-7.6

[1 of 1] Compiling Codec.Archive.Zip ( Codec/Archive/Zip.hs, dist/build/Codec/Archive/Zip.o )

Codec/Archive/Zip.hs:207:19:
    Not in scope: `liftM'
    Perhaps you meant `liftA' (imported from Control.Applicative)

Signed-off-by: Sergei Trofimovich <slyfox@gentoo.org>
---
 Codec/Archive/Zip.hs | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Codec/Archive/Zip.hs b/Codec/Archive/Zip.hs
index ee67691..398ff4a 100644
--- a/Codec/Archive/Zip.hs
+++ b/Codec/Archive/Zip.hs
@@ -68,6 +68,9 @@ import Text.Printf
 import System.FilePath
 import System.Directory ( doesDirectoryExist, getDirectoryContents, createDirectoryIfMissing )
 import Control.Monad ( when, unless, zipWithM )
+#if MIN_VERSION_directory(1,2,0)
+import Control.Monad ( liftM )
+#endif
 import System.Directory ( getModificationTime )
 import System.IO ( stderr, hPutStrLn )
 import qualified Data.Digest.CRC32 as CRC32
-- 
1.8.0.2

